### PR TITLE
Brick game

### LINKED ISSUE #704: BRICK BREAKER GAME GUI
**Is your feature request related to a problem? Please describe.**
brick breaker game gui based

**Additional context**
I would like to do it as a gssoc particiapnt..

### PR DESCRIPTION
**Pull request by Yuvraj-kadale**
# Description
Add brick break game

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
why there are GUI chat application files?

**Comment by Yuvraj-kadale** [comment]
> why there are GUI chat application files?

Oh, I wasn't aware. No clue btw. I have created a new branch and then sent the PR. I will fix this in a while.

File: Brick Breaker game/Readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Brick Breaker Game
+
+Brick Breaker (The game) is a Breakout clonewhich the player must smash a wall of bricks by deflecting a bouncing ball with a paddle. The paddle may move horizontally and is controlled with the side arrow keys.
+
+## Setup instructions
+
+Run `brick_breaker.py`
```
Comment (kaustubhgupta) [inline Brick Breaker game/Readme.md:7]:
```suggestion
Run `python/python3 brick_breaker.py`
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Bugs in game:
- as soon as application window starts, the game starts without any popup or information. Create a welcome page with two options: Start Game and Exit Game
- When the ball is missed by the slider below, there is no popup or warning that the game is over. Implement that and return to the welcome page in case the user loses the game.
- When the game is over, display the final score of the user  (1 point for each brick touched)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- When I click to start the game, the background text should disappear (Click anywhere to start)
- If the user looses the game, display the points scored, and again display "Click anywhere to start)

**Comment by kaustubhgupta** [comment]
@Yuvraj-kadale updates?

**Comment by Yuvraj-kadale** [comment]
> @Yuvraj-kadale updates?

I am out of station.  Will send a PR by tommorow EoD . Will that work?

**Comment by kaustubhgupta** [comment]
> > @Yuvraj-kadale updates?
> 
> I am out of station. Will send a PR by tommorow EoD . Will that work?

sure!

**Review by kaustubhgupta — APPROVED** [review]
Although score thing is still missing, but I will approve it

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..

**Pull request merged**